doc/radosgw/lua-scripting.rst

```diff
@@ -424,3 +428,16 @@ Then, do a restart for the radosgw and upload the following script to the `postR
     assert(s:close())
   end
 
+
+- Trace only for operations of specific bucket
```

Comment:
dont we need the other PR to be merged for the name to be available in preRequest?

Reply:
so. you want to wait for the other PR to be merged first?

Reply:
yes. 
We can move the doc changes to another PR, but IMO this is what makes this PR useful.